stock_indicators/_cslib/__init__.py

```diff
@@ -15,11 +15,9 @@
     load(runtime="coreclr")
     import clr
 except Exception as e:
-    raise ImportError(("fail to import clr.\n"
-    "Stock Indicators for Python has dependency on pythonnet, which uses CLR.\n"
-    "Check that you have CLR installed. It supports .NET 6.0 or above.\n"
-    ".NET:\n\t"
-    "https://dotnet.microsoft.com/download/dotnet")) from e
+    e.add_note("Stock Indicators for Python has dependency on pythonnet, which uses CLR.\n"
+               "Check that you have CLR installed. It supports .NET 6.0 or above.")
```

Comment:
This is a bit confusing, if missing "CLR" capability is only solved by installing .NET SDK, we should make this clearer by simply saying "Install .NET SDK 6.0 or newer to install the required CLR capability." or something like that.

Reply:
> Stock Indicators for Python has a dependency on CLR.  Install .NET SDK 6.0 (or newer) in your environment to add the required CLR capability.

Reply:
LGTM!
[f7ecbe7](https://github.com/DaveSkender/Stock.Indicators.Python/pull/351/commits/f7ecbe7b5df0737b65ba4430089d1eee88ddd362)